designs/administrator/future-openshift/import-multiple-yaml/index.md

```diff
@@ -0,0 +1,38 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Import YAML for multiple resources
+This design details how users can import multiple resources using the **Import YAML** page.
+
+## Design
+
+The **Import YAML** page will still be accessible from the masthead. There are three ways a user can create multiple resources on this page. The instructions on the page should be updated to clarify how users can import YAML: *Drag and drop YAML or JSON files into the editor, or manually enter files and use `- - -` to separate each definition.*
+
+1) Users can manually enter multiple definitions in the editor, using `- - -` to delineate them.
+2) – Users can drag and drop a single YAML with multiple resources defined and separated with `- - -`.
+
+Both methods will show the following in the editor:
+
+![1](img/manual-entry.png)
+
+3) Users can drag and drop multiple files into the YAML area at once.
+
+![2](img/draganddrop.png)
```

Comment:
So just to make sure I understand, this is what the user would see as they are dragging the multiple files onto the editor? Upon release does the 'create' immediately start, or does this page show something about those multiple files and then the user clicks start?

Also this is really small but could this editor support saying "Drop file**s** here" just to further convey they can do many at once?

Reply:
Yes! Upon release, the editor is populated with the contents of the files. I see how that may have been misleading, I will move the image above to be below all the methods and better describe that any method will allow them to view / edit before clicking Create.
Another good catch, will update that image!